package.json

```diff
@@ -4,9 +4,8 @@
   "description": "Standard, sane defaults for logging in slay apps.",
   "scripts": {
     "pretest": "npm run lint",
-    "jscs": "godaddy-js-style-jscs index.js test/*test.js",
-    "eslint": "godaddy-js-style-eslint -c .eslintrc index.js test/*test.js",
-    "lint": "npm run jscs && npm run eslint",
+    "eslint": "eslint --fix index.js test/*test.js",
+    "lint": "npm run eslint",
     "test": "istanbul cover _mocha -- --reporter spec test/*test.js"
```

Comment:
at some point that should probably be `nyc` rather than `istanbul` but can be a separate change/issue.